Docs: Renderer token (#59560)
grafana/grafana: docs/sources/setup-grafana/image-rendering/_index.md

You can see a docker-compose example using a custom configuration file [here](https://github.com/grafana/grafana-image-renderer/tree/master/devenv/docker/custom-config).

### Security

> **Note:** This feature is available in Image Renderer v3.6.1 and later.

You can restrict access to the rendering endpoint by specifying a secret token. The token should be configured in the Grafana configuration file and the renderer configuration file. This token is important when you run the plugin in remote rendering mode. 

> Renderer version v3.6.1 or later require a Grafana version with this feature. These include:

Reviewer: ```suggestion
Renderer versions v3.6.1 or later require a Grafana version with this feature. These include:
```

You can see a docker-compose example using a custom configuration file [here](https://github.com/grafana/grafana-image-renderer/tree/master/devenv/docker/custom-config).

### Security

> > **Note:** This feature is available in Image Renderer v3.6.1 and later versions.

Reviewer: ```suggestion
> **Note:** This feature is available in Image Renderer v3.6.1 and later.
```

> Restrict access to the rendering endpoint by specifying a secret token. The token should be configured in both the Grafana configuration file and the renderer configuration file. This token becomes especially important when running the plugin in remote rendering mode. 

Reviewer: ```suggestion
You can restrict access to the rendering endpoint by specifying a secret token. The token should be configured in the Grafana configuration file and the renderer configuration file. This token is important when you run the plugin in remote rendering mode. 
```

> Renderer version v3.6.1 or later will require a Grafana version where this feature is also available to run, these include:

Reviewer: ```suggestion
Renderer version v3.6.1 or later require a Grafana version with this feature. These include:
```
